x-pack/solutions/security/plugins/security_solution/public/common/links/nav_links.ts

```diff
@@ -42,22 +42,17 @@ export const navLinksUpdater$ = new BehaviorSubject<NavigationLink[]>([]);
 export const navLinks$ = navLinksUpdater$.asObservable();
 
 let currentSubscription: Subscription;
-export const updateNavLinks = (isSolutionNavEnabled: boolean, core: CoreStart) => {
+export const updateNavLinks = (isSolutionNavEnabled: boolean, _core: CoreStart) => {
   if (currentSubscription) {
     currentSubscription.unsubscribe();
   }
   if (isSolutionNavEnabled) {
-    // import solution nav links only when solution nav is enabled
-    lazySolutionNavLinks().then((createSolutionNavLinks$) => {
-      currentSubscription = createSolutionNavLinks$(internalNavLinks$, core).subscribe((links) => {
-        navLinksUpdater$.next(links);
-      });
-    });
-  } else {
-    currentSubscription = internalNavLinks$.subscribe((links) => {
-      navLinksUpdater$.next(links);
-    });
+    return;
```

Comment:
These changes make the landing pages stop working. Should we leave this as is? At least while we are still showing the landing pages.

<img width="938" alt="Captura de pantalla 2025-03-28 a les 12 22 35" src="https://github.com/user-attachments/assets/a9efe49a-1d03-49cd-8011-633c13d7b71b" />